config.py

```diff
@@ -16,4 +16,11 @@
 # 3 = ERROR
 # 4 = SETUP
 # anything else will log nothing
-# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
\ No newline at end of file
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+elevator_can_id: int = 73 # TODO: PLACEHOLDER
+elevator_gear_ratio: float = 0.7 # TODO: PLACEHOLDER
```

Comment:
the elevator_gear_ratio variable should actually be in constants, since that's a physical property that never changes